pkgs/os-specific/linux/kinect-audio-setup/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchgit
+, requireFile
+, pkg-config
+, libusb1
+, p7zip
+}:
+
+let
+  firmwareUrl = "https://download.microsoft.com/download/F/9/9/F99791F2-D5BE-478A-B77A-830AD14950C3/KinectSDK-v1.0-beta2-x86.msi";
```

Comment:
How does the firmware version relate to the version of the package? Where could someone find the latest version? Does the link need to be updated when the package is updated?

Please add a comment that  explains the update procedure, preferably close to the `version` attribute where it is hard to miss.

Reply:
Hi, I left a comment in the source code in this regard.

This is the last known tested firmware by the upstream project. I did not test other firmware revisions as my Kinect v1 microphone worked without any issue on Linux.

Reply:
I see. Thank you for adding the comment.